### ISSUE TITLE
[1.8.9+] various ore .png resource names can be improved to help Resource Packs

### ISSUE THREAD

**Issue by EzerArch** [micdoodle8/Galacticraft#2596]
I'll try to be brief.

1. Some PNG files have **the same file name in the same directory**, the only differentiation is capitalization. This can be a problem when you extract the files locally for development and resource packs.   **[FIXED - radfast :) ]**

E.g.:
`galacticraftplanets\textures\blocks\fluids\AtmosphericGases.png`
`galacticraftplanets\textures\blocks\fluids\atmosphericgases.png`

2. There are **two asset naming systems** present, specially in `galacticraftplanets`. For each `file_name.png` there is still a `fileName.png`.     **[FIXED - radfast :) ]**

Intended? I thought the `camelNaming` became obsolete.

3. (Improvement) [ores occurring](https://wiki.micdoodle8.com/wiki/Ores) in more than one planet, within the same submod (specially in `galacticraftplanets`). I've seen two ways of distinguishing ore variants:

    a) in `galacticraftcore`:
    * ore_copper
    * moonore_copper.png

    b) in `galacticraftplanets`:
    * ore_aluminum.png
    * venus_ore_aluminum.png

For consistency, following the `type_subtype_variant.png` logic, I propose the following:
- ore_iron.png
- ore_iron_mars.png
- ore_iron_asteroids.png
- ore_iron_venus.png

4. BTW, the GalacticraftCore.jar's `galacticraftmoon` dir in is empty.       **[FIXED - radfast :) ]  It was also including empty former folders in the java class structure, e.g. /oxygen.**

**Comment by radfast** [comment]
thanks Ez :) 

this will also be increasing the download size unnecessarily, though I'm not sure by how much

the cause of the issue is git's case-ignore system, git (and Github) are sometimes preserving these files even when we think we deleted them locally.  I'll see if I can clean up some of these.

**Comment by radfast** [comment]
I've looked at this and I think your issues (1) and (2) are **local** issues for you, not actually problems in Galacticraft's repository.  In the [repository](https://github.com/micdoodle8/Galacticraft/tree/master/src/main/resources/assets/galacticraftplanets/textures/blocks) these duplicated files with the old names are not present.

For each branch you're looking at, please try:

    git config core.ignorecase false
    git fetch
    git checkout [branch]
    git clean -f

This should give you a clean copy of the repository (assuming you didn't make any commits locally).  When done, you can set ```git config core.ignorecase true``` to go back to git's default behaviour, if you want it.

(3)  The logic of your proposal looks good, though changes like this have to be made carefully because there will be blockstates and models referencing these.  Could be a few days before I can get to this, as I'm travelling over Easter.

(4)  Thanks, this is probably needs a tweak to ```build.gradle```

**Comment by EzerArch** [comment]
> I've looked at this and I think your issues (1) and (2) are local issues for you

Interesting, but I can see them inside the jar file. (e.g. Galacticraft-Planets-1.10.2-4.0.0.64-BETA.jar\assets\galacticraftplanets\textures\blocks\fluids\)

**Comment by radfast** [comment]
hmm, maybe also "local issues" for the Jenkins then.  that's harder to resolve  :(

**Comment by radfast** [comment]
hmm very strange.  You're right, these duplicate textures are in the Planets 1.10 jar, even though they are not in the 1.8.9 jar.  That's for build 64.   It's making the jar around 1MB larger than it needs to be, which will be a problem when this gets officially released.

I've traced this through the repository and the various internal folders Jenkins uses, and the duplicate textures are not there currently.  My guess is this could be some kind of caching or one-off issue in build 64 which will "fix itself" in build 65.

Let's leave this open until build 65 is out, as a reminder to check this, also your points 3 and 4.

**Comment by radfast** [comment]
I found the caches, and I made sure Jenkins refreshes them so that problems 1, 2 and 4 will all be fixed from next build 65.

_(Note to micdoodle8, I changed Jenkins configuration to rm the ```build/classes``` and ```build/resources``` caches at start of each job.  Now correct for 1.8, 1.10 and 1.11)_


I'll leave this open to remind to rename the ore files, that shouldn't be too hard but I can't do it while travelling.
